pkg/agent/datamodel/sig_config.go

```diff
@@ -229,6 +230,7 @@ var AvailableGen2Distros = []Distro{
 	AKSAzureLinuxV3Arm64Gen2,
 	AKSAzureLinuxV3Arm64Gen2FIPS,
 	AKSAzureLinuxV3CVMGen2,
+	AKSAzureLinuxV3OSGuardGen2FIPSTL,
```

Comment:
do we plan on adding more OSGuard-based distros other than gen2fipsTL in the future?

Reply:
Yes, we'll be adding ARM64 and non-FIPS builds shortly after the x86_64 FIPS version is stood up